sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -519,6 +520,7 @@ static int sof_audio_probe(struct platform_device *pdev)
 
 	/* compute number of dai links */
 	sof_audio_card_rt5682.num_links = 1 + dmic_num + hdmi_num;
+
```

Comment:
remove newline

Reply:
@sathyap-chrome Did you add the newline intentionally? If not, please remove it. Thanks